freertos-rust/src/task.rs

```diff
@@ -134,7 +134,7 @@ impl Task {
         let (success, task_handle) = {
             let name = name.as_bytes();
             let name_len = name.len();
-            let mut task_handle = mem::zeroed::<CVoid>();
+            let mut task_handle = mem::zeroed::<FreeRtosTaskHandle>();
```

Comment:
I wonder if it's necessary (or good) to use `mem::zeroed` here. Could we get away with just setting a null pointer via `core::ptr::null`?

Reply:
Yes, this seems more correct. Done.